roles/install_weston/tasks/recore.yml

```diff
@@ -45,8 +45,5 @@
       WantedBy=multi-user.target
 
 - name: Enable service weston
-  tags: weston
-  service:
-    name: weston
-    enabled: yes
+  shell: systemctl enable weston
```

Comment:
There's a `systemd` module in Ansible you could use here instead: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/systemd_module.html

Reply:
I was getting an error when using the systemd module when running the INSTALL-toggle.yml script. It seems to work well when running using the build-* command.